# GetMove.srv
string input_prompt
---
string current_pos
string final_pos
bool kill_move